tests/test_getting_started.py

```diff
@@ -293,6 +304,9 @@ def test_state_transitions():
     # The save and load test is therefore skipped
     # TestUtils.save_and_load(sol, ocp, False)
 
+    # simulate
+    # TestUtils.simulate(sol, ocp)
```

Comment:
Can you print a warning here? (also for the save_and_load), it is not obvious that it is skipped for this particular condition